src/countries.ts

```diff
@@ -1,70 +1,917 @@
 interface Profile {
-	income: number
+	incomeUSD: number
+	married: boolean
+	oneIncome: boolean
 }
-interface TaxSummary {
+export interface TaxSummary {
 	percentage: number | undefined
+	link?: string
+	notice?: string
 }
 interface TaxStrategy {
-	(profile: Profile): TaxSummary
+	(profile: Profile, exchangeRates: Record<string, number>): TaxSummary
 }
 
 interface TaxStrategies {
 	[name: string]: TaxStrategy
 }
 
-function defaultTaxStrategy() {
-	return { percentage: undefined }
+function defaultTaxStrategy(notice?: string): TaxSummary {
+	return { percentage: undefined, notice }
```

Comment:
This function should implement ` TaxStrategy` interface.
```suggestion
function defaultTaxStrategy(): TaxSummary {
	return { percentage: undefined, notice: 'No data available' }
```